advanced-usage.md

```diff
@@ -390,3 +390,90 @@ auto cb_options_response = cpr::OptionsCallback([](cpr::Response r) {
 ```
 
 Currently, `"PATCH"` is not an implemented HTTP method. It soon will be, and its mechanics will be identitical to the example above. Stay tuned!
+
+## HTTPS Options
+
+In general, you can use the URL of the `https://` protocol directly in your request.
+
+```c++
+auto r = cpr::Get(cpr::Url{"https://www.httpbin.org/get"});
+```
+
+The underlying implementation automatically switches to SSL/TLS protocol if `libcurl` provides the appropriate support.
+
+You can also further customize the behavior of the SSL/TLS protocol by passing in more configuration items in the request. 
```

Comment:
You can also further customize the behaviour of the SSL/TLS protocol by passing more configuration items to the request.